hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -164,6 +199,59 @@ def _default_employee(self):
         compute='_compute_total_time',
         store=True,
     )
+    possible_reviewer_ids = fields.Many2many(
+        string='Possible Reviewers',
+        comodel_name='res.users',
+        compute='_compute_possible_reviewer_ids',
+        store=True,
+    )
+    can_review = fields.Boolean(
+        string='Can Review',
+        compute='_compute_can_review',
+        search='_search_can_review',
+    )
+    complete_name = fields.Char(
+        string='Complete Name',
+        compute='_compute_complete_name',
+    )
+
+    @api.model
+    def fields_view_get(self,
+                        view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        res = super().fields_view_get(
+            view_id=view_id,
+            view_type=view_type,
+            toolbar=toolbar,
+            submenu=submenu,
+        )
+
+        view = etree.XML(res['arch'])
+        self._modify_view_for_policy(view_type, view)
+        res['arch'] = etree.tostring(
+            view,
+            encoding='unicode',
+        ).replace('\t', '')
+
+        return res
+
+    @api.model
+    def _modify_view_for_policy(self, view_type, view):
```

Comment:
@mreficent @astirpe to give you better insights: this method has 4 lines of code specific to validation models.